encrypted-media-respec.html

```diff
@@ -606,7 +606,83 @@
           <section>
             <h5>Get Supported Configuration</h5>
             <p>Given a <a def-id="keysystems"></a> implementation <var>implementation</var>, <a>MediaKeySystemConfiguration</a> <var>candidate configuration</var>, and <var>origin</var>, this algorithm returns a supported configuration or <code>null</code> as appropriate.</p>
-            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.</p>
+            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.
+            </p>
+            <ol>
+              <li>
+                <dl class="switch">
+                  <dt>
+                    If the <a def-id="option-distinctiveIdentifier"></a> member of
```

Comment:
I forgot to reply to this and we are already going down this path, but for the record: I think I agree with the first two paragraphs but think we should keep "optional" for the reasons you mention.